static/src/js/util/request/ousGranuleRequest.js

```diff
@@ -22,6 +22,8 @@ export default class OusGranuleRequest extends CmrRequest {
       'exclude_granules',
       'granules',
       'format',
+      'page_num',
```

Comment:
We should probably get some tests written for this file

Reply:
good catch, didn't realize we didn't have one at all